### ISSUE TITLE
youtube control 0.5.0 center 24 August 2017

### ISSUE THREAD

**Issue by mxyzptlk-113** [inbasic/iyccenter#70]
i'm running Mac OSX 10.12.6. there was an update today and now nothing works. absolutely nothing. 

thought i'd pass this along along with a screen shot of precisely WHAT is doing nothing ;-) 

![youtube control center 0 5 0 2017-08-24 at 6 36 17 pm](https://user-images.githubusercontent.com/31326842/29691923-5c1fa012-88fb-11e7-9d09-02c7ee0bfc9d.png)

**Comment by Funtime60** [comment]
I noticed this on the windows version as well. The new UI is missing most of the options the previous version had. On that note, rolling back to the previous version did not help permanently. It somewhat worked until I updated flash, then it didn't pause one video when I start another video or Auto-pause videos opened in new tabs. I didn't check other features as I never payed attention to them and had no control for them.

**Comment by mxyzptlk-113** [comment]
for what it's worth, i tried all the "features" and got zero response.

usually one addon more, or one less wouldn't be that significant an 
issue; but this is the only plugin that offers those features in one 
place. youtube control center is as irreplaceable as google disconnect, 
and i pray that these problems are just coding errors in a new release. 
hey, may the icon is corrupt ... could be.

i rarely make comments like this, but when i saw what wasn't happening, 
it was important to make it known to the developers asap.


Funtime60 wrote:

**Comment by mybhatt137** [comment]
I'm having the same problems after the 24th August Update. None of the old features seem to work again.
Clicking on the quick settings button on the toolbar just opens another youtube window and does not let you open the settings for the addon. I've tried restarting the browser and all the other remedies from the book. But nothing seems to work. I reckon there is a major mess up in the code. And I wish that all these cluster of problems are resolved in the future.
If someone finds a solution or a bypass to all these problems, do let me know.

**Comment by inbasic** [comment]
0.5.0 is the WebExtension port. I have tried my best to keep features as much as possible.

Note that for 0.5.0 to work you need to have FF >= 54.0

Make sure to reload YouTube tabs and then report bugs. Since I am not sure what the problem is please follow these steps to report

1. Open browser console (Ctrl + Shift + J)
2. Reload the extension (disable and re-enable) -> do you get any new error in the console?
3. Now click on the toolbar button to open a new YouTube page. As soon as a video starts to play, the panel should be accessible. If not, check the error console

**Comment by inbasic** [comment]
Also note that to confirm this is not FF WebEx issue you can test the extension in Chrome as well; https://chrome.google.com/webstore/detail/youtube-control-center/boplfaeblpnpahldaijlikpgdbgdmhko?hl=pt-BR

**Comment by VigorMortis** [comment]
My submission on this same add-on went totally ignored some 26 days ago. Now, of course, I am experiencing total failure with it, a pity. So I removed it and found its predecessor version, installing that one 0.4.3, only to find to my horror that 0.5.0 got installed in its place. Of course, I've removed the wrongly installed latest version now, hoping for some future enabling of the past version to be used. Please let me know.

**Comment by VigorMortis** [comment]
I've also rolled back to Firefox 54.0.1 and have turned off automatic updates. I'd rather wait on this supposed improvement (WebEx) ::holding my nose:: !

**Comment by mxyzptlk-113** [comment]
what OS are you running?

I noticed some changes prior to this update, but it was nothing i 
couldn't work around. i posted the seed for this discussion within 
minutes of the addon's demise. i could never have let this go for almost 
a month. i was honestly hoping that the problem was the OS X 10.12.6 
upgrade, because that would mean that all i had to do was wait until the 
youtube control center caught up with me.

unfortunately, now i will always associate the new favicon with this 
disaster.


VigorMortis wrote:

**Comment by inbasic** [comment]
> I'd rather wait on this supposed improvement (WebEx) 

On FF 55, if there is any feature missing that you depend on please bring it up. There is no support for FF < 55. Note that this is not an improvement. WebEx porting is forced by Mozilla and of course it is less powerful.

> unfortunately, now i will always associate the new favicon with this
disaster.

I am not sure what the problem is with this new release. What exactly do you need that it does not offer?

**Comment by mxyzptlk-113** [comment]
>I am not sure what the problem is with this new release. What exactly 
do you need that it does not offer?

All it presently offers is a bookmark for the opening page of youtube on 
a good day, and on a bad day it doesn't offer anything at all. It used 
to be an incredible search window, where options could be queued up and 
would launch any single choice with a click. There's no longer even a 
search window to type in.

I guess I should be asking you that question. Precisely what should it 
be doing for me? Aside from taking up space on my bookmarks toolbar that 
is ... because that's all it's presently doing, and so far 100% of the 
users on this forum seem to be of a single mind on that issue.



InBasic wrote:

**Comment by inbasic** [comment]
https://www.youtube.com/watch?v=PFHry9xAttk

**Comment by mxyzptlk-113** [comment]
Thank you for supplying the link. You made this choice very simple.

Now that I see how impotent the addon has been rendered, I have no 
trouble uninstalling it. Not sure precisely why you removed all the 
excellent features, but in its present state, it's totally useless for 
my needs.


InBasic wrote:

**Comment by inbasic** [comment]
You can simply switch to the 0.4.3 if you have no plan to move to FF57 when it is released (I am not supporting it though). 

Note that the WebEx is limited in API access so I rewrote the entire extension and added as many features as I could. If there is anything important missing you can bring it up in a new issue. Otherwise blame Mozilla to force its developers to rewrite old and powerful extensions.
